### PR TITLE
add ressource nelmioapidocbundle

### PR DESCRIPTION
**Pull request by n1c01a5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [NgAdminGeneratorBundle](https://github.com/marmelab/NgAdminGeneratorBundle) - Boostrap ng-admin configuration based on `stanlemon/rest-bundle`.
 * [DunglasApiBundle](https://github.com/dunglas/DunglasApiBundle) - Build a REST API which follow Hydra/JSON-LD specification.
 * [API Platform](https://github.com/api-platform/api-platform) - Specialize Symfony edition for the creation of hypermedia REST APIs.
+* [NelmioApiDocBundle](https://github.com/nelmio/NelmioApiDocBundle) - Generates documentation for your REST API from annotations.
```
Comment (jpetitcolas) [inline README.md]: Can you use infinitive form as all other links?

**Comment by jpetitcolas** [comment]
:+1:

**Comment by n1c01a5** [comment]
@jpetitcolas done

**Comment by n1c01a5** [comment]
@jpetitcolas done

**Pull request merged**